### PR TITLE
RS: Enable rack-zone awareness for existing clusters & databases

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/DOC-1969/rs/clusters/configure/rack-zone-awareness/)

DOC-1969 content:
- [Set up rack-zone awareness for an existing cluster](https://docs.redis.com/staging/DOC-1969/rs/clusters/configure/rack-zone-awareness/#existing-cluster)
- [Enable rack-zone awareness for an existing database](https://docs.redis.com/staging/DOC-1969/rs/clusters/configure/rack-zone-awareness/#enable-database-rack-zone-awareness) and [rearrange database shards afterward](https://docs.redis.com/staging/DOC-1969/rs/clusters/configure/rack-zone-awareness/#rearrange-database-shards)

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]
Looks good to me!

File: content/rs/clusters/configure/rack-zone-awareness.md [hunk 1]
```diff
@@ -46,21 +38,101 @@ The rack-zone ID must comply with the following rules:
...
+1. [Update the cluster policy]({{<relref "/rs/references/rest-api/requests/cluster/policy#put-cluster-policy">}}) to enable rack-zone awareness:
+
+    ```sh
+    PUT /v1/cluster/policy
+    { "rack_aware": true }
+    ```
+
+## Enable database rack-zone awareness
+
+Before you can enable rack-zone awareness for a database, you must [set up rack-zone awareness](#set-up-rack-zone-awareness) for the cluster and its nodes.
```
Comment (nermiller) [inline content/rs/clusters/configure/rack-zone-awareness.md:86]:
```suggestion
Before you can enable rack-zone awareness for a database, you must configure rack-zone awareness for the cluster and its nodes. For more information, see [set up rack-zone awareness](#set-up-rack-zone-awareness).
```

File: content/rs/clusters/configure/rack-zone-awareness.md [hunk 2]
```diff
@@ -46,21 +38,101 @@ The rack-zone ID must comply with the following rules:
...
+
+You can use the admin console to enable rack-zone awareness for a database:
```
Comment (nermiller) [inline content/rs/clusters/configure/rack-zone-awareness.md:88]:
```suggestion
To enable rack-zone awareness for a database, in the admin console:
```

File: content/rs/clusters/configure/rack-zone-awareness.md [hunk 3]
```diff
@@ -46,21 +38,101 @@ The rack-zone ID must comply with the following rules:
...
+
+1. From **create database** (for [new databases]({{<relref "/rs/databases/create">}})) or **database** > **configuration** (for existing databases), make sure **Replication** is enabled.
```
Comment (nermiller) [inline content/rs/clusters/configure/rack-zone-awareness.md:90]:
```suggestion
1. Check that **Replication** is enabled in **database > configuration** for existing databases. To create a new database, go to **create database** and use instructions in [Create a Redis Enterprise Software database](({{<relref "/rs/databases/create">}})).
```

File: content/rs/clusters/configure/rack-zone-awareness.md [hunk 4]
```diff
@@ -46,21 +38,101 @@ The rack-zone ID must comply with the following rules:
...
+
+1. Select **Rack-zone awareness**.
+
+1. **Activate** or **Update** your database.
+
+1. For existing databases, you should [rearrange the database shards](#rearrange-database-shards) to ensure your database is optimized for rack-zone awareness.
```
Comment (nermiller) [inline content/rs/clusters/configure/rack-zone-awareness.md:96]:
```suggestion
1. Rearrange database shards to optimize your database for rack-zone awareness. See how in [rearrange the database shards](#rearrange-database-shards).
```

**Review by nermiller — COMMENTED** [review]
Just a couple of edits, Rachel. Thank you for your hard work!

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**